scripts/zkai_native_seq32_attention_mlp_generated_adjacent_label_inventory_gate.py

```
@@ -0,0 +1,818 @@
+        adapter_mode = mode_row["adapter_mode"]
+        variant_id = ADAPTER_MODE_TO_LABEL_ID[adapter_mode]
+        if variant_id not in source_rows or variant_id not in deterministic_rows:
+            raise GeneratedAdjacentLabelInventoryGateError("generated label missing proof accounting")
+        source_row = source_rows[variant_id]
+        policy_row = deterministic_rows[variant_id]
+        if source_row["adapter_mode"] != adapter_mode or policy_row["adapter_mode"] != adapter_mode:
+            raise GeneratedAdjacentLabelInventoryGateError("generated label adapter drift")
+        if policy_row["policy_status"] == "supported_label":
+            proof_accounting_pinned = True
+        elif policy_row["policy_status"] == "rejected_inflating_label":
+            proof_accounting_pinned = True
```

Ruff (0.15.13)

[warning] Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)